frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -4,6 +4,22 @@ import { ClusterServiceVersionKind } from './types';
 import { referenceForProvidedAPI, providedAPIsFor } from './components';
 import * as operatorLogo from './operator.svg';
 
+const isInternal = (crd: { name: string }): boolean => {
+  const internalOpListString = _.get(
+    crd,
+    ['csv', 'metadata', 'annotations', 'operators.operatorframework.io/internal-objects'],
+    '[]',
+  );
+  try {
+    const internalOpList = JSON.parse(internalOpListString); // JSON.parse fails if incorrect annotation structure
+    if (!Array.isArray(internalOpList) || internalOpList.length === 0) return false;
```

Comment:
I don't think this conditional return is necessary. If it's not an array, your catch block should handle the exception. If it is an empty array, the 'some' call will return false.
```suggestion
```